mailer.js

```diff
@@ -390,12 +434,17 @@ module.exports = function (log) {
   Mailer.prototype.postVerifyEmail = function (message) {
     log.trace({ op: 'mailer.postVerifyEmail', email: message.email, uid: message.uid })
 
+    var templateName = 'postVerifyEmail'
     // special utm params, just for this email
     // details at github.com/mozilla/fxa-auth-mailer/issues/110
-    var postVerifyUtmParams = '?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified'
```

Comment:
Consistency is a hobby of mine, so I agree they should all be the same.  Let's keep them all `fx-` so we don't mess up any existing metrics.